### DISCUSSION TITLE
Just wanted to say...

### DISCUSSION THREAD

**Discussion by dyargici** [p2or/blender-loom#60] [General]
Great addon, thanks!  

You are doing gods work and filling holes that should have been filled by the devs years ago (environment variable expansion, hello!)

**Comment by jdknox** [comment]
And I'd like to add:

Thank you also for putting the code into a single python file.  This may sound sarcastic, but it isn't.  With modern editors and IDEs it isn't a problem to jump around in multiple views of the same file, and it makes it _**much**_ easier to search and figure out how the code works instead of being fragmented into a million pointless files buried in 50 nested folders (forgive my _mild_ hyperbole).

I don't know why most of this functionality isn't standard in Blender, but you are doing Poseidon's work out here.

EDIT:
Whoa, _and_ you use [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) date format with proper 24H time format out-of-the-box?!  This just keeps getting better and better.  It's the first time I've _never_ had to tweak all the built-in settings of an add-on...
